packages/@aws-cdk/core/lib/cfn-parse.ts

```diff
@@ -0,0 +1,93 @@
+import { Fn } from './cfn-fn';
+import { Aws } from './cfn-pseudo';
+
+/*
+ * This file contains functions for translating from a pure CFN value
+ * (like a JS object { "Ref": "Bucket" })
+ * to a form CDK understands
+ * (like Fn.ref('Bucket')).
+ */
+
+export function cfnValueToCdkValue(cfnValue: any): any {
```

Comment:
`parseCfnValue`?

Reply:
Sure. Went with `parseCfnValueToCdkValue`, let me know if that works.